ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml

```diff
@@ -0,0 +1,24 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
```

Comment:
`cluster` is not necessary and creates a pitfall, better remove it

Reply:
ack